index.html

```diff
@@ -2653,6 +2653,7 @@ <h3 id="applying">
               </li>
             </ol>
           </li>
+          <li><a>Trim</a>(<var>value</var>) and convert to lower-case</li>
```

Comment:
like Trim is defined, we should define ToLowerCase

ie https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/ToLowerCase